mods/cl_default/locale/default.fr.tr

```diff
@@ -0,0 +1,76 @@
+# textdomain: default
+Furnace is active=Le four est actif
+Furnace is active (@1%)=Le four est actif (@1%)
+Furnace is inactive=Le four est inactif
+Furnace is out of fuel=Le four est à court de combustible
+Furnace is overloaded=Le four est en surcharge
+Stone=Pierre
+Dirt With Grass=Terre avec herbe
+Dirt With Grass and Footsteps=Terre avec herbe et traces de pas
+Dirt=Terre
+Sand=Sable
+Gravel=Gravier
+Sandstone=Grès
+Clay=Argile
+Brick=Brique
+Tree Trunk=Tronc d'arbre
+Jungle Tree Trunk=Tronc d'arbre de la jungle
+Jungle Grass=Herbe de la jungle
+Leaves=Feuilles
+Cactus=Cactus
+Papyrus=Papyrus
+Bookshelf=Bibliothèque
+Glass=Verre
+Fence=Clôture
+Rail=Rail
+Ladder=Échelle
+Stone with Coal=Pierre avec charbon
+Stone with Iron=Pierre avec fer
+Wood=Bois
+Mese=Mese
+Cloud=Nuage
+Water=Eau
+Water Source=Source d'eau
+Lava=Lave
+Lava Source=Source de lave
+Torch=Torche
+Sign=Pancarte
+Some sign=Quelques indications
```

Comment:
this also refers to the in-game sign, so should be "pancarte" too?
(I do not speak french)